epub33/rs/index.html

```diff
@@ -548,6 +548,15 @@ <h3>Spine</h3>
 				<p id="confreq-rs-pkg-spine-unknown" data-tests="#confreq-rs-pkg-spine-unknown">Reading Systems MUST
 					ignore all values expressed in spine <code>itemref</code>
 					<code>properties</code> attributes that they do not recognize.</p>
+
+				<p>
+					When presented with a single <a>Publication Resource</a> that is repeated multiple times in the linear flow 
+					of the spine, Reading Systems SHOULD do their best to display that content in the correct location of that linear flow. 
```

Comment:
That's kind of confusing, as we don't ever mention reordering the spine. It also talks about the linear flow of the spine and then refers back to "that linear flow".

I agree with you in principle about reordering, but I'm not sure the current wording is clear how the "should" relates to the default spine ordering.

Could we maybe flesh this out more like: "When presented with a single Publication Resource that is repeated multiple times in the spine, Reading Systems MUST render that content in each instance where it is referenced when presenting the default linear flow. If the Reading System allows reordering of the spine, it SHOULD attempt to retain a logical ordering of the repeated items relative to their default positions."

I'm not sure that last sentence makes a lot of sense, but maybe it doesn't need to?

Reply:
I don't really want to mention reordering, that was just an example of what we might be making unintentionally illegal. Maybe the problem is we are mandating how this particular content must be displayed, but instead we should be making it clear this content is no different from any other linear content. So something like:

> For rendering purposes when presented with a single Publication Resource that is repeated multiple times in the linear flow of the spine, Reading Systems MUST treat those items as though they were unique item references.

Hmm... that's terrible. But I think we are getting in trouble trying to explain what to do with these, and instead should make clear that whatever a RS does, it should be the same as what they do for correct references.

Reply:
> But I think we are getting in trouble trying to explain what to do with these

Right. Could we maybe generalize it to:

> Reading Systems MUST NOT skip spine references to duplicate manifest item when presenting the linear flow, unless such references are marked as non-linear.

Is that the gist of what we're after?

(No discussion is ever complete without bringing in non-linear content!)

Reply:
That looks good to me! But I am not sure we need to specify the exception for non-linear since it says linear flow, so just `Reading Systems MUST NOT skip spine references to duplicate manifest items when presenting the linear flow.` Unless you want to bring up the fact that "linear flow" is never defined.

Reply:
> Unless you want to bring up the fact that "linear flow" is never defined.

True, and flow gets confusing with the rendering property. It'd be more precise to say "when rendering the linear reading order."

There are a number of other "linear flows" in this PR that should probably be fixed, then, too.